provider/blob/benches/provider_blob.rs

```diff
@@ -0,0 +1,115 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/main/LICENSE ).
+
+use criterion::{black_box, criterion_group, criterion_main, Criterion};
+
+use icu_locid::langid;
+use icu_provider::hello_world::HelloWorldV1Marker;
+use icu_provider::prelude::*;
+use icu_provider_blob::BlobDataProvider;
+
+static BLOB: &[u8] = include_bytes!(concat!(
```

Comment:
Nit: The changes to benches should really be in their own PR. A PR entitled "adding all keys to testdata" should not have earmarks in it.

Reply:
This is a consequence of the hello world removal